### PR TITLE
Front dev

### PR DESCRIPTION
**Pull request by TitovIvan**
Files and folders renamed local but still From Upper Letters Calls in repository

### CODE REVIEW

**Comment by TitovIvan** [comment]
> @TitovIvan please write commit message meaningfully

very good point sir, my bad

File: Templates/Index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<head>
+	<link href="https://allfont.ru/allfont.css?fonts=numans" rel="stylesheet" type="text/css">
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="../Static/CSS/Styles.css">
+</head>
+<body>
+	<header>
+		<img src="../Images/Logo.jpg" class="header_logo" alt="GumApp">
```
Comment (zemags) [inline Templates/Index.html:12]: @TitovIvan please make folder and files names to lowercase
`GumApp`

File: static/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
+html, body, div, span, applet, object, iframe,
```
Comment (zemags) [inline static/css/styles.css:1]: @TitovIvan 
 is it ok, to collect tags like this (huge amount of tags)?
 _need to refactor_

**Review by zemags — CHANGES_REQUESTED** [review]
- [x] change files\folder names to **lowercase**
- [x] remove **private** information from code
- [x] think about huge amount of tags is **styles.css**

File: Templates/Index.html (hunk 1 above)
Comment (TitovIvan) [reply Templates/Index.html:12]: done, earlier think only about folders and files names in local directory, not in code

File: static/css/styles.css (hunk 2 above)
Comment (TitovIvan) [reply static/css/styles.css:1]: this code used for different browsers to make them show content equal (as the same)

**Pull request closed without merging**